src/main/proto/ga4gh/genotype_phenotype_service.proto

```diff
@@ -0,0 +1,202 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "ga4gh/common.proto";
+import "ga4gh/metadata.proto";
+import "ga4gh/genotype_phenotype.proto";
+
+
+/*
+   This protocol defines genotype to phenotype association
+   query system.
+   Suggested ontologies used in type-based queries include:
+   Phenotype:
+   Human Phenotype Ontology (HPO): http://www.ontobee.org/browser/index.php?o=hp
+   Disease Ontology (DO): http://purl.obolibrary.org/obo/DOID_4
+   Feature:
+   Sequence Ontology (SO): http://www.sequenceontology.org/browser/
+   Evidence:
+   Evidence Code Ontology (ECO): http://www.ontobee.org/browser/index.php?o=ECO
+   Ontology for Biomedical Investigations (OBI): http://purl.obofoundry.org/obo/obi/browse
+*/
+service GenotypePhenotypeService {
+
+  // Gets a list of association sets accessible through the API.
+  // `POST /phenotypeassociationsets/search` must accept a JSON version of
+  // `SearchPhenotypeAssociationSetsRequest` as the post body and will return a JSON version
+  // of `SearchPhenotypeAssociationSetsResponse`.
+  rpc SearchPhenotypeAssociationSets(SearchPhenotypeAssociationSetsRequest)
+    returns (SearchPhenotypeAssociationSetsResponse);
+
+  // Gets a list of phenotypes accessible through the API.
+  // `POST /phenotypes/search` must accept a JSON version of
+  // `SearchPhenotypesRequest` as the post body and will return a JSON version
+  // of `SearchPhenotypeAssociationSetsResponse`.
+  rpc SearchPhenotype(SearchPhenotypesRequest)
+      returns (SearchPhenotypesResponse);
+
+  // Gets a list of genotype-phenotype assoications accessible through the API.
+  // `POST /genotypephenotype/search` must accept a JSON version of
+  // `SearchPhenotypesRequest` as the post body and will return a JSON version
+  // of `SearchPhenotypeAssociationSetsResponse`.
+  rpc GetPhenotypeAssociations(SearchGenotypePhenotypeRequest)
```

Comment:
The naming convention should be changed so that search is for search and get is for get.